src/components/Card/CardRangePicker.jsx

```diff
@@ -73,7 +73,6 @@ const CardRangePicker = ({
               [`${iotPrefix}--card--overflow-menuitem-active`]:
                 timeRange === '' || isNil(timeRange),
             })}
-            primaryFocus
```

Comment:
Was focus something that got fixed?

Reply:
This should also have a `selectorPrimaryFocus` for the `OverflowMenu`

Reply:
If it was on the first item it's no longer needed. The menu auto focuses the first element now.